app/commands/exercise/representation/create_search_index_document.rb

```diff
@@ -51,15 +42,13 @@ def passes_latest_tests?
       exists?
   end
 
-  def max_reputation
-    User::ReputationPeriod.where(
-      period: :forever,
-      category: :any,
-      about: :track,
-      track_id: representation.track_id,
-      user_id: representation.published_solutions.select(:user_id)
-    ).maximum(:reputation).to_i
-  end
+  def code = source_submission.files.map(&:content) || []
+  def num_loc = representation.oldest_solution.num_loc
+  def max_reputation = representation.prestigious_solution&.user&.reputation.to_i
```

Comment:
Could this be:

```suggestion
  def num_loc = oldest_solution.num_loc
  def max_reputation = prestigious_solution&.user&.reputation.to_i
```

due to the `delegate:` setup?